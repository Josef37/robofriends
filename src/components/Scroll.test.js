import React from "react";
import { shallow } from "enzyme";
import Scroll from "./Scroll";

describe("Scroll component", () => {
  it("renders correctly with children", () => {
    const wrapper = shallow(<Scroll><div /></Scroll>);
    expect(wrapper.debug()).toMatchSnapshot();
  });

  it("doesn't crash without children", () => {
    const wrapper = shallow(<Scroll />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
