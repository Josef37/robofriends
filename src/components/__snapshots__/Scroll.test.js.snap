// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll component doesn't crash without children 1`] = `"<div style={{...}} />"`;

exports[`Scroll component renders correctly with children 1`] = `
"<div style={{...}}>
  <div />
</div>"
`;
